src/components/CuisineCarousel/index.jsx

```diff
@@ -18,17 +14,29 @@ import classes from "./swiper.module.scss";
 import RestaurantList from "../RestaurantList";
 
 const MyComponent = () => {
```

Comment:
I would name the component and the file name the same. `CuisineCarousel.jsx` 
```suggestion
const CuisineCarousel = () => {
```